src/py/aspen/app/views/auspice.py

```diff
@@ -0,0 +1,48 @@
+import os
+import logging
+logger = logging.getLogger(__name__)
+
+from typing import Iterable, Tuple
+
+import boto3
+from flask import make_response
+from sqlalchemy.orm import joinedload
+
+from aspen.app.app import application
+from aspen.database.connection import session_scope
+from aspen.database.models import (
+    PhyloRun,
+    PhyloTree,
+    WorkflowStatusType,
+)
+
+@application.route("/api/auspice/view/<int:phylo_tree_id>/auspice.json", methods=["GET"])
+def auspice_view(phylo_tree_id: int):
+    with session_scope(application.DATABASE_INTERFACE) as db_session:
+        phylo_runs: Iterable[Tuple[PhyloRun, int]] = (
+            db_session.query(PhyloRun)
+            .options(joinedload(PhyloRun.outputs))
+            .filter(PhyloRun.workflow_status == WorkflowStatusType.COMPLETED)
+        )
+
+        phylo_tree: PhyloTree
+        for phylo_run in phylo_runs:
+            for output in phylo_run.outputs:
+                if isinstance(output, PhyloTree) and output.entity_id == phylo_tree_id:
+                    phylo_tree = output
+                    found = True
+                    break
```

Comment:
same thing that tony mentioned before that this will only break out of the first loop

Reply:
Right yeah.